.github/workflows/main.yml

```diff
@@ -66,9 +66,12 @@ jobs:
     needs: [ shellcheck ]
     strategy:
       matrix:
-        dist: [stretch, buster]
+        dist: [stretch, buster, bullseye]
         arch: [amd64, arm64]
-
+        exclude:
+          # bullseye arm fails to debootstrap with qemu: https://bugs.launchpad.net/ubuntu/+source/qemu/+bug/1928075
+          - dist: bullseye
+            arch: arm64
```

Comment:
How do you plan to keep track of this bug resolution?
cc @beltran-rubo @javsalgar